pages/vi/profiles/gk-web.md

```diff
@@ -0,0 +1,11 @@
+* I am Gautam Khanna situated in Waterloo, ON, Canada {⏰: Eastern Daylight Time(EDT)}
+* I am currently a graduate student of *ECE* in the **University of Waterloo**
+* The OS I am working for is macOS Catalina (Version 10.15)
+* My hobbies are:
+    1. Programming
+       1a. C/C++
```

Comment:
4th markdown style:
Numeric points

You need just one more markdown style added to your profile. Maybe a header? Image? Link? Its up to you.

Good work btw. Almost done with step. Keep up the good work! :smile:

Reply:
Hi, I have added my URL. Please have a look. Thanks!

Reply:
Almost. You can see how to make links [here](https://guides.github.com/features/mastering-markdown/#example-text).

Reply:
Is there no link on the profile?

Reply:
Only as text, not as markdown clickable link.

Reply:
Done

Reply:
Does it take time to see the changes I’ve made?

Reply:
Like the edits I make, do they reflect on the profile after some time?

Reply:
Almost. Taken from the link I sent you above:

Learn more | [here](https://guides.github.com/features/mastering-markdown/)
------------ | -------------
It's very easy to make some words \*\*bold\*\* and other words \*italic\* with Markdown. You can even \[link to Google!\]\(http://google.com\) | It's very easy to make some words **bold** and other words *italic* with Markdown. You can even [link to Google!](http://google.com)

Reply:
Your link is being rendered through html so far.
You'll need to adjust it to be a [markdown link]() in order for it to count.

Reply:
Changed. please have a look.